doc/views_api.md

```diff
@@ -0,0 +1,100 @@
+
+## `StateComponent` API
+
+<pre>
+&lt;class&gt; StateComponent extends Backbone.BaseView
+
+    state: new ProxyState()
```

Comment:
Backbone view are more designed to work with Backbone Models, here we can use `StepModel`.